{% extends "_layout.html" %}

{% block body %}
<div id="fullpart">
  <div id="info">

<h3>Databases</h3>
<h4>Protein Sequence Databases<a href="https://en.wikipedia.org/wiki/List_of_biological_databases"><em>More on Wikipedia</em></a></h4>
<table>

<tr><td><a href="http://www.uniprot.org">UniprotKB </a></td><td>Comprehensive protein knowledgebase, includes manually curated SwissProt and automatically annotated TrEMBL <a href="https://www.ncbi.nlm.nih.gov/pubmed/23161681" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://pir.georgetown.edu/">Protein Information Resource</a></td><td>Integrated public bioinformatics resource to support genomic, proteomic and systems biology research and scientific studies <a href="https://www.ncbi.nlm.nih.gov/pubmed/17569631" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.ncbi.nlm.nih.gov/protein">GenPept</a></td><td>Collection of sequences from several sources, including translations from annotated coding regions in GenBank, RefSeq and TPA, as well as records from SwissProt, PIR, PRF, and PDB <a href="https://www.ncbi.nlm.nih.gov/pubmed/22144687" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.ensembl.org">ENSEMBL</a></td><td>Genome databases for vertebrates and other eukaryotic species <a href="https://www.ncbi.nlm.nih.gov/pubmed/23203987" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
</table>

<h4>Domains and Superfamilies</h4>
<table>

<tr><td><a href="http://www.cathdb.info">CATH/Gene3D </a></td><td>16 million protein domains classified into 2,626 superfamilies <a href="https://www.ncbi.nlm.nih.gov/pubmed/22139938" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="www.ebi.ac.uk/interpro/">InterPro</a></td><td>Classification of proteins into families and prediction of domains and sites.  <a href="https://www.ncbi.nlm.nih.gov/pubmed/22096229" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://pfam.sanger.ac.uk/">PFAM </a></td><td>Collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22127870" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://prodom.prabi.fr">ProDom</a></td><td>Comprehensive set of Protein Domain Families <a href="https://www.ncbi.nlm.nih.gov/pubmed/15608179" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://prosite.expasy.org/">ProSite</a></td><td>Database of Protein Families and Domains <a href="https://www.ncbi.nlm.nih.gov/pubmed/16381852" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://scop.berkeley.edu/">SCOP </a></td><td>Structural Classification of Proteins and Astral <a href="https://www.ncbi.nlm.nih.gov/pubmed/18000004" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://sfld.rbvi.ucsf.edu/">SFLD</a></td><td>Structure-Function Linkage Database: hierarchical classification of enzyme superfamilies.  <a href="https://www.ncbi.nlm.nih.gov/pubmed/22069325" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://supfam.cs.bris.ac.uk/SUPERFAMILY/">SUPERFAMILY</a></td><td>Database of structural and functional annotation for all proteins and genomes <a href="https://www.ncbi.nlm.nih.gov/pubmed/11697912" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://smart.embl-heidelberg.de/">SMART</a></td><td>Allows the identification and annotation of genetically mobile domains and the analysis of domain architectures <a href="https://www.ncbi.nlm.nih.gov/pubmed/22053084" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://meme.nbcr.net/meme/">MEME</a></td><td>Motif-based sequence analysis tools <a href="https://www.ncbi.nlm.nih.gov/pubmed/7584402" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.bioinf.manchester.ac.uk/dbbrowser/PRINTS/index.php">PRINTS</a></td><td>Compendium of protein fingerprints/conserved motifs used to characterize a protein family <a href="https://www.ncbi.nlm.nih.gov/pubmed/22508994" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://bioinfo.mbb.yale.edu/genome" target="_blank" rel="noopener noreferrer">GeneCensus</a> </td><td>Gerstein Lab Resources</tr>
</table>

<h4>Protein Structures and Models<a href="https://en.wikipedia.org/wiki/Protein_structure_database"><em>More on Wikipedia</em></a></h3>
<table>
<tr><td><a href="https://www.wwpdb.org">PDB </a></td><td>Contains information about experimentally-determined structures of proteins, nucleic acids, and complex assemblies <a href="https://www.ncbi.nlm.nih.gov/pubmed/23023942" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="https://salilab.org/modbase/">ModBase </a></td><td>Database of comparative protein structure models generated by the ModPipe comparative modeling pipeline <a href="https://www.ncbi.nlm.nih.gov/pubmed/21097780" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="https://pdb-dev.wwpdb.org/">PDB-Dev</a></td><td>Prototype database of structural models generated by integrative or hybrid methods, such as with our own <a href="https://integrativemodeling.org">IMP</a> software <a href="https://www.ncbi.nlm.nih.gov/pubmed/28877501" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://swissmodel.expasy.org/repository/">SwissModel Repository</a></td><td>Database of annotated three-dimensional comparative protein structure models generated by the fully automated homology-modelling pipeline SWISS-MODEL <a href="https://www.ncbi.nlm.nih.gov/pubmed/18931379" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
</table>

<h4>Miscellaneous</h4>
<table>
<tr><td><a href="https://www.ncbi.nlm.nih.gov/pubmed/" target="_blank" rel="noopener noreferrer">PubMed</a></td><td>Entry page to the NIH databases and tools</td></tr>
<tr><td><a href="http://genome-www.stanford.edu/" target="_blank" rel="noopener noreferrer">Stanford Genomic Resource </a></td><td>Stanford resources maintained by the Saccharomyces Genome Database</td></tr>
<tr><td><a href="http://www.pharmgkb.org/" target="_blank" rel="noopener noreferrer">PharmGKB</a></td><td>Pharmacogenomics Knowledgebase</td></tr>
<tr><td><a href="http://bioinfo.mbb.yale.edu/genome/">GENECENSUS </a></td><td>A hierachical system for comparative genomics study <a href="https://www.ncbi.nlm.nih.gov/pubmed/12384605" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://sbkb.org/">Structural Biology Knowledgebase</a></td><td>Integrates a wide variety of structural biology and structural genomics resources <a href="https://www.ncbi.nlm.nih.gov/pubmed/22527514" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
</table>

<h3>Alignment programs</h3>
<h4>Sequence and structure based sequence alignments <a href="http://en.wikipedia.org/wiki/Sequence_alignment_software"><em>More on Wikipedia</em></a></h4>
<table>
<tr><td><a href="http://www.bioinfo.mpg.de/AlignMe/">AlignMe</a></td><td>Alignment of membrane proteins <a href="https://www.ncbi.nlm.nih.gov/pubmed/23469223" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.clustal.org/">ClustalW</a></td><td>Progressive alignment <a href="https://www.ncbi.nlm.nih.gov/pubmed/18792934" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://prodata.swmed.edu/compass/compass_advanced.php">COMPASS</a></td><td>Multiple Protein sequence alignments with assessment of statistical significance <a href="https://www.ncbi.nlm.nih.gov/pubmed/17517780" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://igs-server.cnrs-mrs.fr/Tcoffee/tcoffee_cgi/index.cgi">EXPRESSO</a></td><td>Multiple sequence alignment of sequences and structures <a href="https://www.ncbi.nlm.nih.gov/pubmed/16845081" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://ffas.burnham.org/ffas-cgi/cgi/document.pl">FFAS</a></td><td>Sensitive profile-profile alignment tool <a href="https://www.ncbi.nlm.nih.gov/pubmed/10716175" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://bioinf.cs.ucl.ac.uk/psipred/">GenTHREADER</a></td><td>Sequence profile and predicted secondary structure <a href="https://www.ncbi.nlm.nih.gov/pubmed/19429599" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://toolkit.lmb.uni-muenchen.de/hhsuite">HHBlits/HHSearch</a></td><td>Homology detection by iterative HMM-HMM comparison <a href="https://www.ncbi.nlm.nih.gov/pubmed/22198341" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.drive5.com/muscle">MUSCLE</a></td><td>Progressive/iterative alignment <a href="https://www.ncbi.nlm.nih.gov/pubmed/15318951" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://zhanglab.ccmb.med.umich.edu/MUSTER">MUSTER</a></td><td>profile-profile alignment <a href="https://www.ncbi.nlm.nih.gov/pubmed/18247410" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://prodata.swmed.edu/promals3d/promals3d.php">Promals3D</a></td><td>Multiple protein sequence/structure alignments using secondary structure prediction, available homologs with 3D structures and user-defined constraints. <a href="https://www.ncbi.nlm.nih.gov/pubmed/18287115" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">PSI-Blast</a></td><td>Position-specific iterative BLAST <a href="https://www.ncbi.nlm.nih.gov/pubmed/9254694" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="https://salilab.org/salign">Salign</a></td><td>Multiple protein sequence/structure alignment <a href="https://www.ncbi.nlm.nih.gov/pubmed/22618536" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://bioinfo3d.cs.tau.ac.il/staccato/">Staccato</a></td><td>Constructs multiple sequence alignment of sets of sequences and structures which is consistent with a structural alignment of the structure set.  <a href="https://www.ncbi.nlm.nih.gov/pubmed/16294339" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.tcoffee.org/">T-Coffee</a></td><td>Sensitive progressive alignment <a href="https://www.ncbi.nlm.nih.gov/pubmed/20205190" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="">MAFFT</a></td><td>Rapid multiple sequence alignment based on fast Fourier transform <a href="https://www.ncbi.nlm.nih.gov/pubmed/15661851" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>

</table>
<h4>Structure <a href="https://en.wikipedia.org/wiki/Structural_alignment_software"><em>More on Wikipedia</em></a></h4>
<table>
<tr><td><a href="http://source.rcsb.org/jfatcatserver/ceHome.jsp">CE</a></td><td>A method for comparing and aligning protein structures <a href="https://www.ncbi.nlm.nih.gov/pubmed/20937596" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="ftp://toolkit.lmb.uni-muenchen.de/hhsearch/">HHSearch</a></td><td>Sensitive pairwise comparison of profile Hidden Markov models <a href="https://www.ncbi.nlm.nih.gov/pubmed/15531603" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://ub.cbm.uam.es/software/mammoth.php">Mammuth</a></td><td>Sequence-independent protein structural alignment method <a href="https://www.ncbi.nlm.nih.gov/pubmed/12381844" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://ub.cbm.uam.es/software/mammothm.php">Mammuth-mult</a></td><td>Multiple alignment version of MAMOTH <a href="https://www.ncbi.nlm.nih.gov/pubmed/15941743" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://bioinfo3d.cs.tau.ac.il/MultiProt">Multiprot</a></td><td>Multiple structural alignments of protein structures by detecting the common geometrical cores between the input molecules <a href="https://www.ncbi.nlm.nih.gov/pubmed/15162494" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="https://salilab.org/salign">Salign</a></td><td>Multiple protein sequence/structure alignment <a href="https://www.ncbi.nlm.nih.gov/pubmed/22618536" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://zhanglab.ccmb.med.umich.edu/TM-align/">TM-align</a></td><td>Highly optimized algorithm for protein structure comparison and alignment <a href="https://www.ncbi.nlm.nih.gov/pubmed/15849316" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://agknapp.chemie.fu-berlin.de/gplus/index.php">GANGSTA+</a></td><td>Protein structure alignment tool and database <a href="https://www.ncbi.nlm.nih.gov/pubmed/18583523" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://bioinfo3d.cs.tau.ac.il/MASS/">MASS</a></td><td>Efficient method for multiple alignment of protein structures and detection of structural motifs. <a href="https://www.ncbi.nlm.nih.gov/pubmed/12855444" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.csse.monash.edu.au/~karun/Site/mustang.html">MUSTANG</a></td><td>Multiple structural alignment algorithm <a href="https://www.ncbi.nlm.nih.gov/pubmed/16736488" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.ebi.ac.uk/msd-srv/ssm/" target="_blank" rel="noopener noreferrer">PDBeFold</a></td><td>3D alignment of protein structures</td></tr>
</table>

<h4>Alignment modules in Molecular Graphics Programs</h4>
<table>
<tr><td><a href="http://www.pymol.org/">PyMOL</a></td><td>Molecular graphics program with many features; use "super" command for sequence-independent alignment <a href="https://www.ncbi.nlm.nih.gov/pubmed/The PyMOL molecular graphics system, SchrÃ¶dinger, 2013" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.cgl.ucsf.edu/chimera">UCSF Chimera</a></td><td>Complex molecular graphics program; for alignments: use MatchMaker tool and command  <a href="https://www.ncbi.nlm.nih.gov/pubmed/15264254" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://accelrys.com/products/discovery-studio/visualization-download.php">Accelrys Discovery Studio Visualization</a></td><td>DS Visualization for Windows and Linux <a href="https://www.ncbi.nlm.nih.gov/pubmed/DS Visualizer, Accelrys Inc. 2013" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://spdbv.vital-it.ch/">Swiss-PDB Viewer</a></td><td>User friendly interface allowing to analyze several proteins at the same time <a href="https://www.ncbi.nlm.nih.gov/pubmed/11465736" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
</table>

<h3>Comparative Modeling and Threading</h3>

<h4>Web-servers <a href="https://en.wikipedia.org/wiki/List_of_protein_structure_prediction_software"><em>More on Wikipedia</em></a></h4>
<table>
<tr><td><a href="http://bmm.cancerresearchuk.org/~3djigsaw/">3D-JIGSAW</a></td><td>Fragment assembly <a href="https://www.ncbi.nlm.nih.gov/pubmed/11835480" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://tardis.nibio.go.jp/fugue/">Fugue</a></td><td>Sequence-structure homology recognition <a href="https://www.ncbi.nlm.nih.gov/pubmed/11419950" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://toolkit.lmb.uni-muenchen.de/hhpred">HHPred</a></td><td>Sensitive protein homology detection and structure prediction by HMM-HMM-comparison <a href="https://www.ncbi.nlm.nih.gov/pubmed/15531603" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/">i-Tasser</a></td><td>Threading fragment structure reassembly <a href="https://www.ncbi.nlm.nih.gov/pubmed/20360767" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.reading.ac.uk/bioinf/IntFOLD/">IntFOLD</a></td><td>Integrated Protein Structure and Function Prediction Server  <a href="https://www.ncbi.nlm.nih.gov/pubmed/21459847" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://manaslu.aecom.yu.edu/M4T/">M4T</a></td><td>Comparative Modelling using a combination of multiple templates and iterative optimization of alternative alignments <a href="https://www.ncbi.nlm.nih.gov/pubmed/17823132" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="https://salilab.org/modweb">ModWeb</a></td><td>Comparative modeling server based on Modeller <a href="https://www.ncbi.nlm.nih.gov/pubmed/21097780" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.sbg.bio.ic.ac.uk/phyre2">Phyre2</a></td><td>Protein homology/analogy recognition engine <a href="https://www.ncbi.nlm.nih.gov/pubmed/19247286" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://raptorx.uchicago.edu/">RaptorX</a></td><td>Remote homology detection, protein 3D modeling, binding site prediction <a href="https://www.ncbi.nlm.nih.gov/pubmed/22814390" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://robetta.bakerlab.org/">Robetta</a></td><td>Rosetta homology modeling and ab initio fragment assembly with Ginzu domain prediction <a href="https://www.ncbi.nlm.nih.gov/pubmed/24035711" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://swissmodel.expasy.org/">SwissModel</a></td><td>Local similarity/fragment assembly <a href="https://www.ncbi.nlm.nih.gov/pubmed/16301204" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
</table>

<h4>Programs<a href="https://en.wikipedia.org/wiki/List_of_protein_structure_prediction_software"><em>More on Wikipedia</em></a></h4>
<table>
<tr><td><a href="ftp://toolkit.genzentrum.lmu.de/pub/HH-suite/">HHSuite</a></td><td>HHSearch, HHBlits, and pre-compiled databases<a href="https://www.ncbi.nlm.nih.gov/pubmed/15531603" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="https://salilab.org/modeller">Modeller</a></td><td>Comparative modeling by satisfaction of spacial restraints <a href="https://www.ncbi.nlm.nih.gov/pubmed/8254673" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://boinc.bakerlab.org/rosetta/">Rosetta@home</a></td><td>Distributed-computing implementation of Rosetta algorithm <a href="https://www.ncbi.nlm.nih.gov/pubmed/4" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="https://www.rosettacommons.org/home">RosettaCM</a></td><td>Molecular modeling softward<a href="https://www.ncbi.nlm.nih.gov/pubmed/24035711" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://dunbrack.fccc.edu/molide/">MolIDE</a></td><td>Open-source cross-platform graphical environment for homology modeling  <a href="https://www.ncbi.nlm.nih.gov/pubmed/18989261" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
</table>

<h4>Miscellaneous</h4>
<table>
<tr><td><a href="https://www.predictprotein.org/" target="_blank" rel="noopener noreferrer">PredictProtein</a></td><td>Secondary structure, families of related proteins, solvent accessibility, transmembrane helix prediction, non-regular secondary structure and intrinsically disordered regions, disulphide bridges and inter-residue contacts, trans-membrane beta barrels structures, and more.  </td></tr>
<tr><td><a href="http://www.molsoft.com/" target="_blank" rel="noopener noreferrer">MolSoft</a></td><td>Large repertoire of tools and databases</td></tr>
</table>


<h3>Evaluation</h3>
<h4>Quality Estimation</h4>
<table>
<tr><td><a href="http://www.cameo3d.org/qe.html">Cameo Quality Estimation</a></td><td>Participating Groups in the Cameo QE project</td></tr>
<tr><td><a href="https://salilab.org/ModEval">ModEval</a></td><td>Estimates model quality by estimating the native overlap, and by computing scores and profile based on statistical potentials: TSVMod, zDOPE, GA341, zDOPE-Profile <a href="https://www.ncbi.nlm.nih.gov/pubmed/21097780" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://proq2.theophys.kth.se/">ProQ2</a></td><td>Support Vector machine based predictor <a href="https://www.ncbi.nlm.nih.gov/pubmed/22963006" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://cms.came.sbg.ac.at/typo3/index.php?id=prosa_download">Prosa 2003</a></td><td>Knowledge-based potentials of mean force. <a href="https://www.ncbi.nlm.nih.gov/pubmed/17517781" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://www.openstructure.org/download/">QMEAN Local</a></td><td>Composite scoring function describing five different structural descriptors <a href="https://www.ncbi.nlm.nih.gov/pubmed/17932912" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://swissmodel.expasy.org/workspace/index.php?func=tools_structureassessment1">SwissModel Workspace</a></td><td>Several assessment tools to estimate the quality and structural features of protein models and template structures: Dfire, Procheck, DSSP, Promotif, Anolea, Gromos, QMEANS6 <a href="https://www.ncbi.nlm.nih.gov/pubmed/16301204" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://nihserver.mbi.ucla.edu/Verify_3D/">Verify 3D</a></td><td>Measures the compatibility of a protein model with its sequence using a 3D profile. <a href="https://www.ncbi.nlm.nih.gov/pubmed/1853201" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://swift.cmbi.ru.nl/gv/whatcheck/">What_Check</a></td><td>Protein verification tools </td></tr>
<tr><td><a href="http://www.reading.ac.uk/bioinf/ModFOLD/ModFOLD_form_2_0.html" target="_blank" rel="noopener noreferrer">ModFold</a></td><td>Model Quality Assessment Server</td></tr>
<tr><td><a href="https://prosa.services.came.sbg.ac.at/prosa.php" target="_blank" rel="noopener noreferrer">ProSA-web</a></td><td>Protein-Structure Analysis</td></tr>
<tr><td><a href="http://kinemage.biochem.duke.edu/" target="_blank" rel="noopener noreferrer">Kinemage</a></td><td>3D Macromolecule analysis, includes Kinemage 3D graphics, Rampage for Ramachandran plot generation, MolProbity structure validation, and more</td></tr>
</table>

<h4>Methods evaluation</h4>
<table>
<tr><td><a href="http://predictioncenter.org/">CASP</a></td><td>Critical Assessment of protein Structure Prediction for Template-based modeling and free modeling, bi-annual <a href="https://www.ncbi.nlm.nih.gov/pubmed/21997831" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
<tr><td><a href="http://cameo3d.org/">CAMEO</a></td><td>Continuous automated model evaluation of accuracy and reliability of various protein structure prediction services in a fully automated manner <a href="https://www.ncbi.nlm.nih.gov/pubmed/23624946" target="_blank" rel="noopener noreferrer"><img  src="//salilab.org/img/pubmed_light.png"></a></td></tr>
</table>
<h3>Protein-protein Complexes </h3>
<table>
<tr><td><a href="http://bioinfo3d.cs.tau.ac.il/PatchDock/" target="_blank" rel="noopener noreferrer">PatchDock</a></td><td>Molecular docking algorith based on shape complementarity principles</td></tr>
<tr><td><a href="http://structure.pitt.edu/servers/fastcontact/" target="_blank" rel="noopener noreferrer">FastContact</a></td><td>Free energy scoring tool for protein-protein complex structures</td></tr>
<tr><td><a href="http://bioinfo3d.cs.tau.ac.il/FiberDock/" target="_blank" rel="noopener noreferrer">FiberDock</a></td><td>Flexible Induced-fit Backbone Refinement in Molecular Docking </td></tr>
<tr><td><a href="https://genesilico.pl/toolkit/unimod?method=MetaMQAPII" target="_blank" rel="noopener noreferrer">MetaMQAPII</a></td><td>Bioinformatics and protein engineering toolkit</td></tr>
</table>

<h3>Protein-Ligand Resources</h3>
<table>
<tr><td><a href="http://blaster.docking.org/" target="_blank" rel="noopener noreferrer">DockBlaster</a></td><td>A free virtual docking server</td></tr>
<tr><td><a href="http://zinc.docking.org/" target="_blank" rel="noopener noreferrer">Zinc</a></td><td>A database of commercially-available compounds for virtual screening</td></tr>
</table>


<h3>Comparative Genomics and Interaction Networks</h3>
<table>
<tr><td><a href="http://www.cgl.ucsf.edu/cytoscape/structureViz/" target="_blank" rel="noopener noreferrer">structureViz</td><td>Cytoscape plugin that links visualization of biological networks in Cytoscape with visualization and analysis of molecular structures in UCSF Chimera.</td></tr>
<tr><td><a href="http://www.microbesonline.org/" target="_blank" rel="noopener noreferrer">MicrobesOnline</a></td><td>Comparative and functional genomics analysis</td></tr>
<tr><td><a href="http://string-db.org/" target="_blank" rel="noopener noreferrer">String</a></td><td>Protein function, and interaction network analysis</td></tr>
</table>


<h3>Miscellaneous</h3>
<table>
<tr><td><a href="http://pedant.gsf.de/" target="_blank" rel="noopener noreferrer">Pedant3</a></td><td>Protein Extraction, Description and ANalysis Tool</td></tr>
<tr><td><a href="http://workbench.sdsc.edu/" target="_blank" rel="noopener noreferrer">Biology Workbench</a></td><td>Web-based tool for biologists</td></tr>
</table>

</div></div> <!-- end div fullpart -->
{% endblock %}
