{% extends "_layout.html" %}

{% block body %}
<div id="fullpart">
            <h3>Our Resources</h3>
            <h5>&nbsp;<br>Databases</h5>
          <table>
          <tr>
              <td> <a href="/DBAli/">DBAli</a> </td>
            <td> Database of sequence and structure alignments generated by different alignment methods </td>
          </tr>
          <tr>
            <td> <a href="http://www.nysgrc.org/nysgrc/icedb.html">IceDB</a></td>
            <td>Integrated database for <a href="http://www.nysgrc.org/"> New York Structural Genomics Research Consortium</a></td>
          </tr>
          <tr>
            <td nowrap> <a href="http://www.salilab.org/~libusha/pnas2008/34pssmdb.txt">Integral Membrane Database</a> </td>
            <td> Integral membrane database for 34 genomes. </td>
          </tr>
          <tr>
            <td> <a href="/ligbase/">LigBase</a></td>
              <td>Database of structurally defined ligand binding sites in PDB</td>
          </tr>
          <tr>
            <td> <a href="http://www.salilab.org/~libusha/pnas2008/memb_counts.txt">Membrane Protein counts</a> </td>
            <td> Counts of membrane protein families represented in 34 genomes.  </td>
          </tr>
          <tr>
            <td> <a href="/modbase/"> ModBase</a> </td>
            <td> Comprehensive database of annotated comparative protein structure models </td>
          </tr>
          <tr>
            <td> <a href="/pibase/"> PiBase</a> </td>
            <td> Comprehensive database of structurally defined interfaces in proteins </td>
          </tr>
          <tr>
            <td> <a href="https://salilab.org/ftp/albase-3/"> ALBASE-3</a> </td>
            <td> Database of multiple protein structure alignments used to derive MODELLER restraints </td>
          </tr>
          <tr>
            <td> <a href="/decoys">Structural decoys</a> </td>
            <td> Series of structural decoys used in the Sali Lab.<br> Includes the MOULDER, MODPIPE and MODEM sets.  </td>
          </tr>
          <tr>
            <td align="left"> <a href="/models/models.html"> Models and Alignments in our publications</a> </td>
            <td> Links to models and alignments in our publications </td>
          </tr>
          </table>
          <h5>&nbsp;<br>Web Services</h5>
          <table>
          <tr>
            <td> <a href="/allosmod/">AllosMod</a> </td>
            <td>Modeling of Ligand-induced Protein Dynamics and Beyond </td>
          </tr>
          <tr>
            <td> <a href="/allosmod-foxs/">AllosMod-FoXS</a> </td>
            <td>Structure Generation and SAXS Profile Calculations </td>
          </tr>
          <tr>
            <td> <a href="/~eva/">EVA</a> </td>
            <td> Server for continuous, automatic, and statistically significant assessment of protein structure prediction servers </td>
          </tr>
          <tr>
            <td> <a href="/foxs"> FoXS</a> </td>
            <td> Fast SAXS profile computation with Debye formula</td>
          </tr>
          <td> <a href="/foxsdock"> FoXSDock</a> </td>
            <td> Macromolecular docking restrained by a small angle X-ray scattering profile</td>
          </tr>
          <tr>
            <td> <a href="/ligscore"> LigScore</a> </td>
            <td> Pose &amp; Rank - a web server for scoring protein-ligand complexes</td>
          </tr>
          <tr>
            <td nowrap> <a href="/evaluation/">ModEval</a> </td>
            <td> Web server to model evaluation using TSVMod, DOPE and GA341</td>
          </tr>
          <tr>
            <td> <a href="/modloop/"> ModLoop</a> </td>
            <td> Server for modeling of loops in protein structures </td>
          </tr>
          <tr>
            <td> <a href="/modweb"> ModWeb</a> </td>
            <td> Server for automated comparative protein structure modeling </td>
          </tr>
          
          <tr>
            <td> <a href="/peptide"> PCSS</a> </td>
            <td> Predicts peptide recognition specificity based on sequence and structure features</td>
          </tr>
          <tr>
            <td> <a href="/saxsmerge"> SAXS Merge</a> </td>
            <td>An automated statistical method to merge SAXS profiles from different concentrations and exposure times</td>
          </tr>
          <tr>
            <td> <a href="http://sanchezlab.org/servers/pg/"> pG server</a> </td>
            <td> Program for model assessment based on the ProsaII Z-score </td>
          </tr>
          </table>
          <h5>&nbsp;<br>Programs and Methods</h5>
          <table>
          <tr>
            <td> <a href="/Flex-EM">Flex-EM</a> </td>
            <td>A method for fitting and refining a component structure within its map at intermediate resolution</td>
          </tr>
          <tr>
            <td> <a href="//integrativemodeling.org/">IMP</a> </td>
            <td>Integrative Modeling Platform</td>
          </tr>
          <tr>
            <td> <a href="/idock/">iDOCK</a> </td>
            <td>The integrative docking method uses atomistic models of two interacting proteins and one or more datasets from experimental techniques (SAXS, EM, NMR and MS)</td>
          </tr>
          
          <tr>
            <td> <a href="/modeller/"> MODELLER</a> </td>
            <td> Program for comparative protein structure modelling by satisfaction of spatial restraints </td>
          </tr>
          <tr>
            <td> <a href="/mdt"> MDT</a> </td>
            <td> MDT prepares a raw frequency table, given information from MODELLER alignments and/or PDB files </td>
          </tr>
          <tr>
            <td> <a href="http://pibase.janelia.org/modtie/v1.11/index.html">ModTie</a> </td>
            <td> MODTIE makes structure-based predictions of binary and higher-order protein complexes utilizing ModBase and PiBASE.</td>
          <tr>
            <td> <a href="/SOAP"> SOAP</a> </td>
            <td>Statistically optimized atomic statistical potentials for ranking loops and protein-protein interfaces.</td>
          </tr>
          <tr>
            <td> <a href="/modview/modview2.shtml"> ModView</a> </td>
            <td> Web plugin for visualization and analysis of multiple protein sequences and structures </td>
          </tr>
          <tr>
            <td> <a href="/asgl/asgl.html"> ASGL</a> </td>
            <td> Program for preparing simple PostScript plots </td>
          </tr>
          <tr>
            <td> <a href="https://salilab.org/ftp/miscellaneous_programs/"> Miscellaneous programs</a> </td>
            <td> PSA (surface accessibility), ELLIPSOID (antigenicity index) </td>
          </tr>
          <tr>
            <td> <a href="/patcher/"> Patcher</a> </td>
            <td> Program for localizing binding sites in protein structures based on a patch optimization scheme and a composite scoring function </td>
          </tr>
          </table>

         <tr><td>&nbsp;</td></tr>
        </table>            
            
            
            </div> <!-- end div fullpart -->
{% endblock %}
